Firestore/Source/Local/FSTLevelDBUtil.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#include <memory>
+
+@class FSTPBTargetGlobal;
+
+namespace leveldb {
+class DB;
+struct ReadOptions;
+}
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FSTLevelDBUtil : NSObject
```

Comment:
I'm not a super fan of this kind of "util" classes like these. They become a dumping ground of unrelated things.

Meanwhile all the code that really wants to manipulate this proto and save it in other circumstances is in the `FSTLevelDBQueryCache` already. I'd prefer we just leave this code in there, perhaps just making it available as a static method.

Reply:
Sure, although I think it's weird for the migrations to depend on the query cache. I guess I'll move the read options to FSTLevelDB, since it definitely doesn't make sense to have a util class w/ one method.

Reply:
Ok, I shuffled some things around.